spec/text_message_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'text_message'
+
+describe TextMessage do
+
+  describe "#send_order_text" do
+    it 'can send text message to display order information' do
+      ENV = {account_sid:'double',auth_token:'double',phone_number:'double'}
+      text = TextMessage.new
+      order = double(:fake_order)
+      allow(text).to receive(:send_order_text).and_return(nil)
+      class Twilio
+      end
+      class Rest < Twilio
+      end
+      class Client < Rest
+        def initialize(arg1,arg2)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `arg1`. If it's necessary, use `_` or `_arg1` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.<br>Lint/UnusedMethodArgument: Unused method argument - `arg2`. If it's necessary, use `_` or `_arg2` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.